### ISSUE TITLE
We haven't CSV files (e.g., for RISM, etc.) for reconciliation. 

### ISSUE THREAD

**Issue by candlecao** [DDMAL/linkedmusic-datalake#226]
Therefore, I will try reconciling RDF (n-triple) files on OpenRefine

**Comment by candlecao** [comment]
@ahankinson Typically, we should reconcile the entities and properties with those in Wikidata.
Has RISM already been reconciled? (I haven't reviewed the details of the RISM RDF file you shared yet.)

**Comment by ahankinson** [comment]
Some have been reconciled, yes. 

https://rism.online/people/2539 -- bottom of the page, "Other standard identifier"

**Comment by lpugin** [comment]
I once tried to have a sparql query listing all the wiki data institutions with a link to RISM but not in the other direction. I don't think I got it working but we can look again. Would that be helpful?

**Comment by fujinaga** [comment]
Yes, that would be great. We use OpenRefine for reconciliations.

**Comment by candlecao** [comment]
> https://rism.online/people/2539

I saw it. Thank you!

**Comment by candlecao** [comment]
> Actually I could not get this from sparql since the information is not yet in the endpoint. Attached two files listing what we have in Muscat.
> 
> [institutions.csv](https://github.com/user-attachments/files/17992901/institutions.csv) [people.csv](https://github.com/user-attachments/files/17992903/people.csv)
> 
> However, WikiData will have more than this: https://w.wiki/CHJt https://w.wiki/CHJy

Thanks. I will look into them.

**Comment by candlecao** [comment]
This is solved and the reconciled files are stored in path ArchiveForReconciledEntries > RISM of our datalake.
Thank you again @lpugin
